proposals/target-typed-static-member-lookup.md

```diff
@@ -183,6 +185,27 @@ For an invocation expression such as `e(...)` where the invoked expression `e` i
 
 Even though the conversion always succeeds when the invoked expression `e` is a  _target-typed member binding expression_, further errors may occur if the invocation expression cannot be bound for any of the same reasons as though the _target-typed member binding expression_ was a non-target-typed expression, qualified as a member of `T`. For instance, the member might not be invocable, or might return a type other than `T`.
 
+### Target-typing after the `new` keyword
+
+A core scenario for this proposal is enable the `new` operator to look up nested derived types. This provides symmetry between production and consumption of values with class DUs and with today's type hierarchies based on nested derived classes or records.
+
+```cs
+new .Case1(arg1, arg2)
+```
+
+This balances the consumption syntax:
+
+```cs
+du switch
+{
+    .Case1(var arg1, var arg2) => ...
+}
+```
+
+This would continue to be target-typed static member access (since nested types are members of their containing type), which is distinct from target-typed `new` since a definite type is provided to the `new` operator.
+
+TODO: spec the conversion
```

Comment:
There are other todos in this file. I'm happy merging as-is, since it doesn't seem like there will be an issue in principle coming up with the conversion, and the proposal is being discussed at a higher level currently.